Makefile

```diff
@@ -3,6 +3,7 @@
 VERSION ?= 1.10
 
 .inline-deps:
+	git clean -fdx
```

Comment:
This appeared to fix the mrandersoning, probably it'd be better to improve the `clean` task